markdown/events/2022/bytesize-gitpod.md

```diff
@@ -25,3 +25,121 @@ It is our hope that these talks / videos will build an archive of training mater
+
+[3:06](https://www.youtube.com/watch?v=kBoC6QBU-M0t=186)
+To get started we may as well just dive in, so that we can show exactly how this works. If you want to do this at another time, it might be difficult to do it as well at the same time but if you want to then go ahead. All you need is the browser and your GitHub credentials and to actually open any GitHub URL as a Gitpod environment. Then you just need to add this prefix that's shown here before the GitHub repository URL or you can install a Gitpod browser extension. I will put all these in the notes afterwards. You can download this extension and it will basically add a Gitpod button, a green button, to every Git repository and all it's doing is just making this prefix before the URL, so that you can quickly access with one click. Especially if you're already logged in with your GitHub credentials, then it will be one click and you'll get straight into the working repository. Finally I just wanted to mention that this is actually being done through a file in the Git repository called `.gitpod.yml` file and this is the file that tells Gitpod what particular pieces of code we need and how to set up that environment so that the particular code that we're interested in works.
+
+[4:29](https://www.youtube.com/watch?v=kBoC6QBU-M0t=269)
+We're going to just go straight into the live demo. I'll explain exactly what it looks like when we go into this environment, how you can add extensions into the environment. You can add different add-ons. This is a picture from Gitpod. You can add Docker or you can add lots of different functionality within your containerized environments to test your code. I will explain how to use Git within this and also opening files like PDFs etc and live rendering for website development.
+
+[5:02](https://www.youtube.com/watch?v=kBoC6QBU-M0t=302)
+What we're going to do now is, we're just going to go straight into how to use Gitpod. Hopefully everyone can see this. The example pipeline we're going to use is the nf-core test pipeline. I've already got the browse extension so I actually have this link already. I could also copy the link address, open up a new tab and just paste it in. But for speed I'll just click on it. So when you log in, all you need to do... and at this point it would ask you for your credentials. For me it's not because I'm already logged in. This is going straight from that repository into Gitpod. Along on the other side you can see the explorer set says, these are all the files within your repository; you have a window here to look at visualization of code or PDFs etc and then you have a terminal at the bottom. You also should have extra buttons here which I can't see. Anyway, there's all the things you would normally want to do on your own computer, you can also do here.
```

Comment:
```suggestion
What we're going to do now is, we're just going to go straight into how to use Gitpod. Hopefully, everyone can see this. The example pipeline we're going to use is the nf-core test pipeline. I've already got the browse extension so I actually have this link already. I could also copy the link address, open up a new tab and just paste it in. But for speed, I'll just click on it. So when you log in, all you need to do... and at this point, it would ask you for your credentials. For me, it's not because I'm already logged in. This is going straight from that repository into Gitpod. On the other side you can see the explorer set says, these are all the files within your repository; you have a window here to look at visualization of code or PDFs etc and then you have a terminal at the bottom. You also should have extra buttons here which I can't see. Anyway, there's all the things you would normally want to do on your own computer, you can also do here.
```